src/app/Fake.DotNet.AssemblyInfoFile/AssemblyInfoFile.fs

```diff
@@ -213,7 +213,6 @@ module AssemblyInfoFile =
 
     /// Creates a C# AssemblyInfo file with the given attributes and configuration.
     /// The generated AssemblyInfo file contains an AssemblyVersionInformation class which can be used to retrieve the current version no. from inside of an assembly.
-    [<System.Obsolete "Please use 'create' instead">]
```

Comment:
You don't like create anymore or what is the rationale?

Reply:
I thought I was a big agressive on the obsolete.  What if someone wanted to control themselves?
Since we don't plan to delete, only make private i see no harm in leaving without obsolete

Reply:
> Since we don't plan to delete

;)